lib/ip_wrangler/main.rb

```diff
@@ -69,10 +65,10 @@ def check_released_resource(released_resource)
 # List NAT port(s) for specified private IP
 get '/nat/port/*' do |private_ip|
   sandbox do
-    if valid_ip? private_ip
+    if valid_ip?(private_ip)
       $nat.get_nat_ports(private_ip).map { |nat_port|
         puts nat_port[:public_ip]
-        nat_port[:public_ip] = $config[:ext_ip]
+        nat_port[:public_ip] = $config['ext_ip']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.